docs/hello.rst

```diff
@@ -65,16 +63,35 @@ You can also reuse the package management system, Python IDEs, and existing Pyth
 Portability
 -----------------
 
-Taichi supports both CPUs and NVIDIA GPUs.
+Taichi code can run on CPUs or GPUs. Depends on your platform:
 
 .. code-block:: python
 
-  # Run on GPU
+  # Run on NVIDIA GPU, require CUDA installed
   ti.init(arch=ti.cuda)
+  # Run on GPU, with OpenGL backend
+  ti.init(arch=ti.opengl)
+  # Run on GPU, with Metal backend, if you're on OS X
```

Comment:
Metal backend -> the Metal backend
you're -> you are (In general, avoid `I'll`, `you're`, `can't` in formal writing)